manager.py

```diff
@@ -55,43 +71,121 @@ def configure(self, context):
         self.logger.info('REST Manager set to {}'.format(self._rest_manager))
```

Comment:
Might want to remove this, it isn't that helpful

Reply:
I agree